package.json

```diff
@@ -38,51 +15,41 @@
     "test:chrome:headless-chrome": "npm run test -- --test-suite chrome-new-headless",
     "test:chrome:headful": "npm run test -- --test-suite chrome-headful",
     "prepublishOnly": "npm run build",
-    "prepare": "node typescript-if-required.js && husky install",
+    "install": "npm run install:husky && lerna run install",
+    "install:husky": "husky install",
     "lint": "run-s lint:prettier lint:eslint",
     "lint:prettier": "prettier --check .",
     "lint:eslint": "([ \"$CI\" = true ] && eslint --ext js --ext ts --quiet -f codeframe . || eslint --ext js --ext ts .)",
-    "install": "node install.js",
-    "generate:sources": "tsx utils/generate_sources.ts",
-    "generate:artifacts": "tsx utils/generate_artifacts.ts",
     "generate:markdown": "tsx utils/generate_docs.ts",
     "format": "run-s format:*",
     "format:prettier": "prettier --write .",
     "format:eslint": "eslint --ext js --ext ts --fix .",
     "docs": "run-s build generate:markdown",
-    "debug": "npm run build:dev && mocha --inspect-brk",
+    "debug": "npm run build:test && mocha --inspect-brk",
     "commitlint": "commitlint --from=HEAD~1",
-    "clean": "rimraf lib && rimraf test/build",
-    "check": "run-p check:*",
-    "check:protocol-revision": "tsx scripts/ensure-correct-devtools-protocol-package",
+    "clean": "lerna run clean && rimraf **/.wireit",
+    "check": "lerna run check && run-p check:*",
     "check:pinned-deps": "tsx scripts/ensure-pinned-deps",
-    "build": "npm run build:prod",
-    "build:dev": "run-s generate:sources build:tsc:dev && run-p bundle:third_party generate:artifacts",
-    "build:prod": "run-s generate:sources build:tsc:prod && run-p bundle:third_party generate:artifacts",
-    "build:tsc:dev": "tsc -b test",
-    "build:tsc:prod": "tsc -b tsconfig.lib.json",
-    "bundle:third_party": "rollup --config rollup.third_party.config.js"
+    "build": "lerna run build",
+    "build:test": "wireit"
   },
-  "files": [
-    "lib",
-    "install.js",
-    "typescript-if-required.js",
-    "!**/*.tsbuildinfo"
-  ],
-  "author": "The Chromium Authors",
-  "license": "Apache-2.0",
-  "dependencies": {
-    "cross-fetch": "3.1.5",
-    "debug": "4.3.4",
-    "devtools-protocol": "0.0.1036444",
-    "extract-zip": "2.0.1",
-    "https-proxy-agent": "5.0.1",
-    "progress": "2.0.3",
-    "proxy-from-env": "1.1.0",
-    "rimraf": "3.0.2",
-    "tar-fs": "2.1.1",
-    "unbzip2-stream": "1.4.3",
-    "ws": "8.9.0"
+  "wireit": {
```

Comment:
is it essential to have wireit? The home page says it's alpha level.

Reply:
Wireit vastly improved build times and is very much necessary for to help with building. It's similar to my job framework I built, but more automated. At some point, all the commands should migrate to this.

Reply:
Hey @aomarks, I actually have a feature request. It'd be nice to have phony targets.

Reply:
Cool! Can you describe how a phony target would work? (Feel free to open an issue to discuss further)

Reply:
A phony target is basically a target with no corresponding npm command. This would help with organising commands and prevent people from running "intermediate" commands.

Reply:
Additionally, it would make sense to have a .wireitrc or a wireit.config.js since we are polluting package.json which will add additional bytes for library users.

Reply:
Hey @jrandolf! The phony targets feature discussed above is now available in wireit 0.9.4: https://github.com/google/wireit/blob/main/CHANGELOG.md#094---2023-01-30